src/components/ProgramList.tsx

```diff
@@ -0,0 +1,165 @@
+
+    return (
+        <Center>
+            <Wrap spacing="50px" justify="space-between">
+                {imagesAndDescriptions.map((item, idx) => {
+                    return item.name == "placeholder" ? (
+                        <WrapItem
+                            flexBasis="300px"
+                            flexGrow={1}
+                            display="hidden"
+                            key={idx}
+                            _hover={{
+                                borderColor: "gray.600",
+                                borderWidth: 1,
+                            }}
+                        ></WrapItem>
+                    ) : (
+                        <WrapItem
+                            flexBasis="300px"
+                            key={idx}
+                            flexGrow={1}
+                            _hover={{
+                                borderColor: "gray.600",
+                                borderWidth: 1,
+                            }}
+                        >
+                            <Box borderWidth="1px" width="100%">
+                                <AspectRatio width="100%" ratio={4 / 2}>
+                                    <Image
+                                        src={item.image}
+                                        boxSize="200"
+                                        key={idx}
+                                        data-index={idx}
+                                        fit="cover"
+                                        alt={item.name}
+                                    />
+                                </AspectRatio>
+
+                                <Box p="6">
+                                    <Box d="flex" alignItems="baseline">
+                                        <Box
+                                            mt="1"
+                                            fontWeight="semibold"
+                                            as="h3"
+                                            lineHeight="tight"
+                                            isTruncated
+                                        >
+                                            {item.name}
+                                        </Box>
+                                        <Spacer />
+                                        <Badge
+                                            borderRadius="full"
+                                            padding="1"
+                                            textTransform="capitalize"
+                                            fontWeight="medium"
+                                            letterSpacing="wide"
+                                            backgroundColor="gray.600"
+                                            color="white"
+                                        >
+                                            {item.tag}
+                                        </Badge>
+                                        <Badge
```

Comment:
If these Badges are supposed to be the same then maybe you can create a const for them with all of the style settings so that any changes are consistent? it's not really necessary if there are only 2 badges for each program but if a program can have multiple tags then it might be useful

e.g. `const CardTag = (props) = <Badge {style props here}>{props.children}</Badge>`

Reply:
Also on that note, it might be nice to add a bit of fade to the hover? @stacykwok @christinaruu thoughts?

idk how hard that is to do LOL but I feel like chakra probably has a built in style prop for that, if not maybe it can be done separately so that we can try to merge this by sunday?

Reply:
Should we do this after the meeting on sunday? 
I also thought maybe we can make a badge component that already has all the style stuff, if all badges will have the same css

Reply:
Fade in/fade out would be nice to have, but only if there's enough time!

Reply:
Fade in/out would be nice to have, but only if there's enough time!